charts/partials/templates/_noop.tpl

```diff
@@ -6,10 +6,10 @@ name: noop
 resources:
   limits:
     cpu: "50m"
-    memory: "10Mi"
+    memory: "25Mi"
   requests:
     cpu: "50m"
-    memory: "10Mi"
+    memory: "25Mi"
```

Comment:
Looks good. Not sure if it was discussed recently, but we could also just re-use `proxyInit`'s resources if we wanted to make it configurable and avoid adding additional configuration parameters.

Reply:
I do like that idea in general. But I'm also OK shipping this without using configuration